src/lib/components/info-card/index.tsx

```diff
@@ -16,28 +16,37 @@ export const InfoCard: FC<PropsWithChildren<InfoCardProps>> = ({
 
   return (
     <Box
-      p="m"
-      width={width ?? '15rem'}
-      height="7.5rem"
+      px="m"
+      py="xl"
+      bg="onPrimary"
+      height="8.25rem"
```

Comment:
try remove this `height` and put a `gap="l or m"` (according theme-space to 1rem)